cmd/klotho/dependencies.go

```diff
@@ -95,8 +96,8 @@ func installPulumi(ctx context.Context) error {
 }
 
 func isDockerInstalled() bool {
-	// Check if docker is installed
-	cmd := exec.Command("docker", "--version")
+	// Check if docker is installed and the daemon is running
+	cmd := exec.Command("docker", "ps")
```

Comment:
I don't think we actually invoke this anywhere right now, but the intent here is to let you know you need to install docker, not verify that it's running.

Reply:
The only reason you'd need docker is to do a build, which requires the daemon running.

Reply:
It's moot since we haven't fully implemented the check, but with `ps`, we'd just need to clarify the message.